### ISSUE TITLE
Arrow Meta IDEA plugin: publish snapshot versions

### ISSUE THREAD

**Issue by rachelcarmena** [arrow-kt/arrow-meta#321]
Previous step for: #245

**Comment by rachelcarmena** [comment]
I take a note about the last issue: `Arrow Meta not loaded: since build 193.5662 > IC-192.7142.36`

**Comment by rachelcarmena** [comment]
This line is included in `plugin.xml` when building the artifact:

```
<idea-version since-build="193.5662" until-build="193.*"/>
```

**Comment by rachelcarmena** [comment]
According Kotlin IDEA plugin, that line would have to include:

```
<idea-version since-build="193.3793.14" until-build="201.*"/>
```

It it's included in `plugin.xml` won't be calculated. Let's see how it can be updated.

**Comment by rachelcarmena** [comment]
We're going to use Kotlin repository data. Those values are here: https://github.com/JetBrains/kotlin/blob/v1.3.61/idea/resources/META-INF/plugin.xml.193 (tree for `v1.3.61`)

**Comment by rachelcarmena** [comment]
Solved via #397

**Comment by rachelcarmena** [comment]
It works ok. Now we have this repository to install IDEA plugin: https://meta.arrow-kt.io/idea-plugin/snapshots/1.3.61-SNAPSHOT/updatePlugins.xml

However, the artifact doesn't include the shadowed compiler plugin artifact:  #398

**Comment by rachelcarmena** [comment]
It's not necessary to include the shadowed compiler plugin artifact into the artifact. 

Distribution file must be provided instead of that artifact.

It's solved via #577
